pysisyphus/calculators/DFTD3.py

```diff
@@ -0,0 +1,64 @@
+from pysisyphus.calculators.Calculator import Calculator
+import re
+import numpy as np
+import os.path
```

Comment:
Import is not needed

Reply:
Imports should be ordered like this

1. Modules from stdlib
2. Blank line
3. Third party packages, e.g., numpy
4. Blank line
5. Imports from own package, i.e., pysisyphus
6. Two blank lines

```python
import re

import numpy as np

from pysisyphus.calculators.Calculator import Calculator
```